integration_test/metadata/integration_metadata.go

```diff
@@ -218,20 +223,62 @@ func NewIntegrationMetadataValidator() *validator.Validate {
 	return v
 }
 
-func AssertMetric(metric *ExpectedMetric, series *monitoringpb.TimeSeries) error {
+func AssertMetric(expectedMetric *ExpectedMetric, series *monitoringpb.TimeSeries) error {
 	var err error
-	if series.ValueType.String() != metric.ValueType {
-		err = multierr.Append(err, fmt.Errorf("valueType: expected %s but got %s", metric.ValueType, series.ValueType.String()))
+	if series.ValueType.String() != expectedMetric.ValueType {
+		err = multierr.Append(err, fmt.Errorf("valueType: expected %s but got %s", expectedMetric.ValueType, series.ValueType.String()))
 	}
-	if series.MetricKind.String() != metric.Kind {
-		err = multierr.Append(err, fmt.Errorf("kind: expected %s but got %s", metric.Kind, series.MetricKind.String()))
+	if series.MetricKind.String() != expectedMetric.Kind {
+		err = multierr.Append(err, fmt.Errorf("kind: expected %s but got %s", expectedMetric.Kind, series.MetricKind.String()))
 	}
-	if !SliceContains(metric.MonitoredResources, series.Resource.Type) {
-		err = multierr.Append(err, fmt.Errorf("unexpected monitored_resource: expected %v but got %s", metric.MonitoredResources, series.Resource.Type))
+	if !SliceContains(expectedMetric.MonitoredResources, series.Resource.Type) {
+		err = multierr.Append(err, fmt.Errorf("unexpected monitored_resource: expected %v but got %s", expectedMetric.MonitoredResources, series.Resource.Type))
 	}
-	err = multierr.Append(err, assertMetricLabels(metric, series))
+	err = multierr.Append(err, assertMetricLabels(expectedMetric, series))
 	if err != nil {
-		return fmt.Errorf("%s: %w", metric.Type, err)
+		return fmt.Errorf("%s: %w", expectedMetric.Type, err)
+	}
+	if len(series.Points) == 0 {
+		err = multierr.Append(err, fmt.Errorf("Metric %s has at least one data points in the time windows", expectedMetric.Type))
```

Comment:
Should this error message be something like ? 

```
fmt.Errorf("Metric %s should have at least one data point in the time window", expectedMetric.Type)
```

Also, in my understanding this can't really happen right ? Like getting an empty time series from a queried metric. In any case, it is good to have the check here.

Reply:
I will address this comment on my next PR.